### ISSUE TITLE
Feature: show intensity value in grid mode

### ISSUE THREAD

**Issue by tlambert03** [napari/napari#2955]
## 🚀 Feature
Not sure how hard this would be, but it would be nice to be able to get the intensity value of _any_ pixel when hovering in grid mode.  Currently: you will only see the pixel value if you happen to hover over the selected layer in the grid... but since there's no overlapping layers in grid mode, it should in theory be possible for all layers that support `get_value`?

**Comment by jni** [comment]
Yeah, this ties into my desire (which apparently I never made into an issue, according to a quick search 😅) to show the value of the first opaque layer, rather than the selected layer, in the status bar. In my experience it's very confusing to show the value of a currently-obstructed layer in the status bar. But maybe that's just me?

Anyway, solving my issue automatically solves yours. 😊

**Comment by tlambert03** [comment]
Agreed, I like yours better!

**Comment by psobolewskiPhD** [comment]
With https://github.com/napari/napari/pull/7597 merged, grid mode layers can overlap, so this isn't so simple.
Also, since this was opened, we merged https://github.com/napari/napari/pull/7584 which makes the coordinates for grid mode layers a bit more intuitive.

**Comment by jni** [comment]
> this isn't so simple

I *think* it's no less simple than before? The implementation is to iterate over all visible layers from top to bottom and run get_status on it, until you're on a layer.

Of course, this will be slower than doing a single layer. However, that's less of a problem now that the computation was moved to a separate thread, and, in grid mode in particular, you can use layer extents to quickly move on to the next layer.

**Comment by jni** [comment]
The 0.6.0 milestone is aspirational because I *think* it's feasible, and if it is then it would be another good headliner. But I may bump it to 0.6.1. [Programmers' Credo](https://www.globalnerdy.com/2021/06/07/the-programmers-credo/) and all that. 😅

**Comment by jni** [comment]
This is the bit of code that would need updating, I think:

https://github.com/napari/napari/blob/9891e006aec227cbe1bc3e544e288c38c6d45ede/napari/components/viewer_model.py#L578-L604
